packages/gatsby/src/utils/webpack.config.js

```diff
@@ -229,6 +229,15 @@ module.exports = async (
             plugins.eslintGraphqlSchemaReload(),
           ])
           .filter(Boolean)
+        if (process.env.GATSBY_EXPERIMENTAL_DEV_SSR) {
+          const MiniCssExtractPlugin = require(`mini-css-extract-plugin`)
+          // Don't use the default mini-css-extract-plugin setup as that
+          // breaks hmr.
+          configPlugins.push(
+            new MiniCssExtractPlugin({ filename: `[name].css` })
+          )
+          configPlugins.push(plugins.extractStats())
+        }
```

Comment:
Can we instead remove `filename` / `chunkFilename` from default options in https://github.com/gatsbyjs/gatsby/blob/e563bda658d01496f564bf789e37f776c0cf42d3/packages/gatsby/src/utils/webpack-utils.ts#L701-L706

and pass those from `build-javascript` case + pass `` filename: `[name].css` `` from `develop` case?

It makes easier (in my mind at least :P) to find when something is added if the code goes through same abstraction everywhere (if we already uise `webpack-utils`, then we should use it always)

Also - we can do ``configPlugins.push(plugins.extractText({ filename: `[name].css }), plugins.extractStats())`` in single go (`.push` support arbitrary number of args) (tho minor NIT and also doesn't apply just when going through `extractText` helper)

Reply:
👍